testdata/results/result.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonResult
+metadata:
+  name: result
+spec:
+  targetNamespace: openshift-pipelines
+  logs_api: true
+  log_level: debug
+  db_port: 5432
+  db_host: tekton-results-postgres-service.openshift-pipelines.svc.cluster.local
+  logging_pvc_name: tekton-logs
+  logs_path: /logs
+  logs_type: File
+  logs_buffer_size: 2097152
+  auth_disable: false
```

Comment:
Let's disable it for now, it will be much easier to write further tests for results